src/main/java/randoop/condition/OutcomeTable.java

```diff
@@ -0,0 +1,109 @@
+package randoop.condition;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import randoop.test.ExpectedExceptionGenerator;
+import randoop.test.ExtendGenerator;
+import randoop.test.InvalidCheckGenerator;
+import randoop.test.PostConditionCheckGenerator;
+import randoop.test.TestCheckGenerator;
+
+/** Records the outcome of checking all of the conditions for a method. */
+public class OutcomeTable {
+
+  /** Indicates whether this table is empty. */
+  private boolean isEmpty = true;
+
+  /** Indicates whether this table has a valid entry. */
+  private boolean hasValid = false;
+
+  /** The list of throws clauses for this table. */
```

Comment:
This Javadoc comment doesn't say much:  it gives the type, and says it's "for this table", which is obvious.  It would be more helpful to say, for example, that their preconditions were all satisfied.